packages/components/src/components/tag/tag-styles.spec.ts

```diff
@@ -60,27 +37,29 @@ xdescribe('getColors()', () => {
     ['dark', false],
   ])('should return correct css for theme: %s and hasInvertedTheme: %s', (theme, hasInvertedTheme) => {
     jest.spyOn(tagUtils, 'hasInvertedThemeColor').mockReturnValue(hasInvertedTheme);
-    expect(getColors(getThemedColors('light'), 'background-surface', theme)).toMatchSnapshot();
+
+	const themedColors = getThemedColors(theme);
+    expect(getColors(themedColors, 'background-surface', theme)).toMatchSnapshot();
   });
 });
 
-xdescribe('getComponentCss()', () => {
+describe('getComponentCss()', () => {
   it.each<Parameters<typeof getComponentCss>>([
     ['background-default', true, 'light'],
     ['background-default', false, 'light'],
     ['neutral-contrast-high', true, 'light'],
     ['notification-success', true, 'light'],
     ['notification-warning', true, 'light'],
     ['notification-error', true, 'light'],
-    ['notification-neutral', true, 'light'],
+    ['notification-information', true, 'light'],
```

Comment:
`notification-neutral` should be deprecated but still being supported and mapped to `notification-information` for backwards compatibility reasons.

Reply:
Changed it - now we support both